src/main/java/org/eluder/coveralls/maven/plugin/util/Md5DigestInputStream.java

```diff
@@ -39,6 +39,6 @@ public Md5DigestInputStream(final InputStream stream) throws NoSuchAlgorithmExce
     }
 
     public String getDigestHex() {
-        return DatatypeConverter.printHexBinary(getMessageDigest().digest());
+        return new String(Hex.encodeHex(getMessageDigest().digest(),false));
```

Comment:
```suggestion
        return new String(Hex.encodeHex(getMessageDigest().digest(), false));
```
Can this be merged then?